### DISCUSSION TITLE
Does PyART have the slice option for the grid objects

### DISCUSSION THREAD

**Discussion by syedhamidali** [ARM-DOE/pyart#1476] [Q&A] [answered]
Hi,

I want to subset or slice any field or all the fields of the grid object. Does Pyart have the functionality for that? I know we can convert the grid to xarray dataset and then slice it, but I want the grid object to be sliced. How can I do that either by using x and y values or lat, lon values

**Comment by mgrover1** [comment, marked as answer]
This functionality does not currently exist, and I would suggest working with the xarray dataset object for the time being, otherwise a PR here would be welcome! Or an open issue!
